docs/patterns/components/Table/index.js

```diff
@@ -162,14 +163,24 @@ const BasicTable = () => (
             >
               Email
             </TableHeader>
+            <TableHeader scope="col">Actions</TableHeader>
           </TableRow>
         </TableHead>
         <TableBody>
           {sortedData.map(contact => (
-            <TableRow>
+            <TableRow key={contact.email}>
               <TableCell>{contact.first_name}</TableCell>
               <TableCell>{contact.last_name}</TableCell>
               <TableCell>{contact.email}</TableCell>
+              <TableCell>
+                <IconButton
+                  icon="trash"
+                  label="Delete"
+                  onClick={() => {
+                    console.log(`Delete ${contact.email}`);
+                  }}
+                />
+              </TableCell>
```

Comment:
Initially, I just did this for testing, but I thought it was nice to leave in there.